docs/src/__examples__/CallOutBanner/DEFAULT.tsx

```diff
@@ -26,8 +26,21 @@ export default {
       }
     />
   ),
-  info: {
-    title: "Default callout banner",
-    description: "Callout banners use illustrations and titles to draw attention to actions.",
-  },
+  exampleKnobs: [
+    {
+      component: "CallOutBanner",
+      knobs: [
+        {
+          name: "title",
+          type: "text",
+          defaultValue: "Accommodation in Warsaw",
+        },
+        {
+          name: "description",
+          type: "text",
+          defaultValue: "Select the perfect place to rest during your stay in Warsaw.",
+        },
+      ],
+    },
+  ],
```

Comment:
In this example a similar problem happens where there the CallOutBanner loses an image as soon as the playground is open, but it should look the same.